### PR TITLE
reordered and added coveralls info

### LINKED ISSUE #271: add the coverall requirements to the contribution.md

### PR DESCRIPTION
**Pull request by SamMarkowitz**
Fixes #271 
Signed-off-by: Sam Markowitz <sam.markowitz@hp.com>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -21,6 +21,33 @@ When a pull request fails testing, the author is expected to update the pull req
...
+
+##Conduct
+
+Whether you are a regular contributor or a newcomer, we care about making this community a safe place for you.
+
+We are committed to providing a friendly, safe and welcoming environment for all regardless of their background and the extent of their contributions.
+Please avoid using nicknames that might detract from a friendly, safe and welcoming environment for all.
+Be kind and courteous.
+Those who insult, demean or harass anyone will be excluded from interaction. In particular, behavior that excludes people in socially marginalized groups will not be tolerated.
+We welcome discussion about creating a welcoming, safe and productive environment for the community. If you have any questions, feedback or concerns please let us know. (info@openscore.io)
```
Comment (orius123) [inline CONTRIBUTING.md]: do we want to use this email?
does it even work?

**Comment by orius123** [comment]
@SamMarkowitz ?

**Comment by SamMarkowitz** [comment]
Don't know if the email address is used. @orius123 Do you think I should change it to a different one (support@cloudslang.io) or remove it all together?

**Comment by oritstone** [comment]
@orius123 WDYT?

**Comment by orius123** [comment]
I would go with changing it to the new support mail you mentioned

**Pull request merged**